llvm/lib/Target/DirectX/DXILOpLowering.cpp

```diff
@@ -105,20 +105,46 @@ class OpLowerer {
     return false;
   }
 
-  [[nodiscard]]
-  bool replaceFunctionWithOp(Function &F, dxil::OpCode DXILOp) {
+  struct ArgSelect {
+    enum class Type {
+      Index,
+      I8,
+      I32,
+    };
+    Type Type = Type::Index;
+    int Value = -1;
```

Comment:
The default value here `int Value = -1` is different than the default value of the other defined `ArgSelect`.

Reply:
Both are -1 now, matching the tablegen too.